### PR TITLE
Add Dockerfile.pi for building docker image for raspberry pi

### PR DESCRIPTION
**Pull request by tl-nguyen**
## Summary
Add Dockerfile.pi for building docker image on raspberry pi

## Quick changelog

- Add Dockerfile.pi
- Add requirements-pi.txt

## What's new?
Deploying freqtrades bots on Raspberry Pi with docker

### CODE REVIEW

File: Dockerfile.pi [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ENV LD_LIBRARY_PATH /usr/local/lib
+
+# Install berryconda
+RUN wget https://github.com/jjhelmus/berryconda/releases/download/v2.0.0/Berryconda3-2.0.0-Linux-armv7l.sh \
+ && bash ./Berryconda3-2.0.0-Linux-armv7l.sh -b \
+ && rm Berryconda3-2.0.0-Linux-armv7l.sh
+
+# Install dependencies
+COPY requirements.txt /freqtrade/
+RUN sed -i -e '/^numpy==/d' -e '/^pandas==/d' -e '/^scipy==/d' ./requirements.txt \
```
Comment (xmatthias) [inline Dockerfile.pi]: Can we create a `requirements_pi.txt` which does not list these requirements?

That would be the easier (and cleaner) solution as it'll work for people not running on docker too -  and if it's aded to `.pyup.yml` they are updated automatically as well.

Comment (tl-nguyen) [reply Dockerfile.pi]: @xmatthias I didn't know about .pyup.yml, that's why :D. I will add it back,  thanks!

**Comment by xmatthias** [comment]
I did not forget this - but i need to find the time to power up my PI and actually test this and i'm currently tight on time (should get to it latest on the weekend).

It looks good from a code perspective however :+1:

**Comment by hroff-1902** [comment]
I also have got a spare RPI in the dusty chest, what can be the sequence of actions?

(If you provide instructions, that should then go to the docs...)

**Comment by tl-nguyen** [comment]
> I also have got a spare RPI in the dusty chest, what can be the sequence of actions?
> 
> (If you provide instructions, that should then go to the docs...)

Hey, first you have to install qemu emulator
https://www.qemu.org/download/

And then you can build the image for arm architecture:
```
docker build -t yourdockerhubname/freqtrade.pi -f Dockerfile.pi .
```
And after that. ssh to your RPI and pull down the image, and run it just like you would run with the x86 docker image.

I've made a arm image from my Mac and pushed it here: tlnguyen/freqtrade.pi:develop . Works fine when I pulled and ran on my RPI

PS: if for some reason the build fails. The instructions [here](https://blog.hypriot.com/post/setup-simple-ci-pipeline-for-arm-images/) suggest running this before the build:
```
docker run --rm --privileged multiarch/qemu-user-static:register --reset
```

**Comment by hroff-1902** [comment]
Is Raspbian from Noobs okay for running docker with freqtrade? (I.e., what are the system reqs? 

(Last time my RPI was booted it was running custom Android built from sources, I don't think that's of one of the target platforms for docker on RPI...)

**Comment by tl-nguyen** [comment]
> Is Raspbian from Noobs okay for running docker with freqtrade? (I.e., what are the system reqs?
> 
> (Last time my RPI was booted it was running custom Android built from sources, I don't think that's of one of the target platforms for docker on RPI...)

Ya, use the Lite version. I'm able to run 4+ strategies with RPI 3 B+ with the Lite version. There's an easier way to install raspbian by using PiBakery. You can change hostname, password add wifi username/password etc.

**Pull request merged**